src/components/ex-download/ex-download.scss

```diff
@@ -45,28 +49,35 @@
   }
   &.download-preview {
     .download_item {
-      @apply bg-gray-3 pb-6;
+      @apply pb-6;
+      background: #f5f8fa;
+
       @screen sm {
         max-width: 376px;
       }
       .download_link {
-        @apply text-s flex flex-wrap justify-center text-center;
+        @apply text-s text-gray-1 flex flex-wrap justify-center text-center;
         .download_image {
           @apply inline-flex;
           max-width: 200px;
         }
         .download_title {
-          @apply text-s items-center justify-center w-full mt-2 mb-0 font-semibold no-underline;
+          @apply text-s text-gray-1 items-center justify-center w-full mt-2 mb-0 font-semibold no-underline;
+
+          &:hover {
+            @apply text-primary;
```

Comment:
Something is wrong with the underline. The underline stays gray on mouse-over. Should also turn blue:

![image](https://user-images.githubusercontent.com/88790097/134164519-c50d4995-291d-477e-b6a4-844b8263d0f2.png)

Reply:
Should be fixed now